.github/workflows/ci_docker.yml

```diff
@@ -0,0 +1,85 @@
+name: CI docker
```

Comment:
I think you should rename this file to be consistent with the event warehouse since we are going to be adding more workflow files with the migration off travis.

Reply:
Ok renamed